src/uniprotkb/config/UniProtKBColumnConfiguration.tsx

```diff
@@ -609,6 +607,22 @@ UniProtKBColumnConfiguration.set(UniProtKBColumn.ccSequenceCaution, {
   },
 });
 
+UniProtKBColumnConfiguration.set(UniProtKBColumn.featureCount, {
+  label: 'Features',
+  render: (data) => {
+    const counts = data?.extraAttributes?.countByFeatureType;
+    return (
+      counts && (
+        <ExpandableList displayNumberOfHiddenItems>
+          {Object.keys(counts)
+            .sort()
```

Comment:
Would it be more interesting to sort by count? Maybe ask Marija, I don't have an strong opinion and I don't know how users use this column

Reply:
Had a chat with Marija and she's advised to leave alphabetical for now as it's unclear how users use the column: compare between rows or do they just want to know the most numerous. Marija has has added to the future testing list.